frontend/src/individual_form.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-import { InputAdornment, makeStyles, TextField } from "@material-ui/core";
+import { InputAdornment, StylesProvider, TextField } from "@material-ui/core";
```

Comment:
VSCode marks the `StylesProvider` as unused in this file. Are you sure the import is needed?